bundles/azure-service-broker-0.0.1/chart/azure-service-broker/charts/redis/README.md

```diff
@@ -0,0 +1,250 @@
+# Redis
+
+[Redis](http://redis.io/) is an advanced key-value cache and store. It is often referred to as a data structure server since keys can contain strings, hashes, lists, sets, sorted sets, bitmaps and hyperloglogs.
```

Comment:
```suggestion
[Redis](https://github.com/helm/charts/tree/master/stable/redis) is an advanced key-value cache and store. It is often referred to as a data structure server since keys can contain strings, hashes, lists, sets, sorted sets, bitmaps and hyperloglogs.
```